ui/src/App.tsx

```diff
@@ -221,7 +231,7 @@ const App = () => {
       recurring,
       frequency,
       recurrenceBegins,
-      recurrenceEnds,
+      recurrenceEndUtc,
```

Comment:
Could you explain the decision to change `reccurenceEnds` to have `Utc` prepended and no `recurrenceBegins` as well?